gsa/src/web/pages/extras/__tests__/cvsscalculatorpage.js

```diff
@@ -0,0 +1,189 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+import {act} from 'react-dom/test-utils';
+
+import {fireEvent, rendererWith, waitForElement} from 'web/utils/testing';
+
+import CvssCalculator from 'web/pages/extras/cvsscalculatorpage';
+
+/* eslint-disable no-console */
+
+// this is just a little hack to silence a warning that we'll get until we
+// upgrade to 16.9: https://github.com/facebook/react/pull/14853
+const originalError = console.error;
+beforeAll(() => {
+  console.error = (...args) => {
+    if (/Warning.*not wrapped in act/.test(args[0])) {
+      return;
+    }
+    originalError.call(console, ...args);
+  };
+});
+
+afterAll(() => {
+  console.error = originalError;
+});
+
+/* eslint-enable no-console */
+
+const calculateScoreFromVector = jest.fn().mockReturnValue(
+  Promise.resolve({
+    data: 7.5,
+  }),
+);
+
+const gmp = {
+  cvsscalculator: {
+    calculateScoreFromVector,
+  },
+  settings: {
+    manualUrl: 'http://docs.greenbone.net/GSM-Manual/gos-5/',
+  },
+  user: {
+    renewSession: jest.fn().mockReturnValue(
+      Promise.resolve({
+        data: 'foo',
+      }),
+    ),
+  },
+};
+
+const location = {
+  query: {cvssVector: 'AV:N/AC:L/Au:N/C:P/I:P/A:P'},
+};
+
+describe('CvssCalculator page tests', () => {
+  test('Should render with default values', () => {
+    const {render} = rendererWith({
+      gmp,
+      store: true,
+    });
+
+    const {element, getAllByTestId} = render(<CvssCalculator />);
+
+    expect(element).toMatchSnapshot();
```

Comment:
```suggestion
```

Reply:
Done :) And removed snapshot + its folder